europe/README.md

```diff
@@ -1 +1,2 @@
 # EUROPE
+- [Iceland](./Iceland/README.md)
```

Comment:
There's that README file under Iceland directory missing. If you add it, then the PR will pass the tests.